src/expressions/refExpression.ts

```diff
@@ -150,6 +162,7 @@ export class RefExpression extends Expression {
     js.name = this.name;
     if (this.nest) js.nest = this.nest;
     if (this.type) js.type = this.type;
+    if (this.ignoreCase !== null && this.ignoreCase !== undefined) js.ignoreCase = this.ignoreCase;
```

Comment:
an accepted shorthand for `(this.ignoreCase !== null && this.ignoreCase !== undefined)` is `(this.ignoreCase != null)` this is the only time using `!=` (vs `!==`) is acceptable and our linter respects this.